packages/hadron-build/lib/download-center.js

```diff
@@ -1,198 +1,54 @@
+/* eslint-disable mocha/no-exports */
 const fs = require('fs-extra');
-const debug = require('debug')('mongodb-download-center');
-const AWS = require('aws-sdk');
+const { DownloadCenter } = require('@mongodb-js/dl-center');
 
+const DOWNLOADS_BUCKET = 'downloads.10gen.com';
 const MANIFEST_BUCKET = 'info-mongodb-com';
+const MANIFEST_OBJECT_KEY = 'com-download-center/compass.json';
 
-let maybeUpload = (CONFIG, asset) => {
-  const bucket = 'downloads.10gen.com';
-  const key = `${CONFIG.download_center_key_prefix}/${asset.name}`;
-
-  const params = {
-    Bucket: bucket,
-    Key: key,
-    Body: fs.createReadStream(asset.path),
-    ACL: 'public-read',
-    Metadata: {
-      Name: asset.name
-    }
-  };
-
-  return new Promise((resolve, reject) => {
-    new AWS.S3({
-      params: {
-        Bucket: bucket,
-        Key: key
-      }
-    }).headObject( (err, data) => {
-      if (data) {
-        debug(`Asset at s3://${bucket}/${key} already exists.  Skipping.`);
-        return resolve(data);
-      }
```

Comment:
Before we were skipping uploads for existing files, I removed this check because this just doesn't sound right to me: if we would need to re-run asset publishing (which is separate from making assets listed on downloads page) I think we would want the latest version to be available. But please tell me if this doesn't make sense to y'all